script/test

```diff
@@ -14,13 +14,6 @@ echo "==> Updating..."
 script/update
 
 echo "==> Linting files..."
-if [ -n "$1" ]; then
-  # pass arguments to standard. This is useful for checking a single JavaScript file, 
-  # or a directory using the glob pattern. You can also pass in --fix to fix 
-  # lint errors.
-  npx standard "$@"
-else
-  npx standard
-fi
+npm run lint
```

Comment:
How does `npm run lint` know not to care about the bundled scripts in `src/assets/js`? It doesn't complain about them, just curious why not 🤔 (maybe they're compliant with Standard)

Reply:
Good question! Just found this in the docs, which is nice! [https://standardjs.com/#how-do-i-ignore-files](https://standardjs.com/#how-do-i-ignore-files)

> Certain paths (node_modules/, coverage/, vendor/, *.min.js, and files/folders that begin with . like .git/) are automatically ignored.
> 
> **Paths in a project's root .gitignore file are also automatically ignored.**

Reply:
Nice!